nav2_dwb_controller/nav_2d_utils/test/2d_utils_test.cpp

```diff
@@ -0,0 +1,162 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2018, Locus Robotics
```

Comment:
Are you locus robotics? :-)

Reply:
Sorry. I had just copy pasted that part from another file. I have corrected this.